force-app/main/default/classes/ContactInformationSettingsVMapper.cls

```diff
@@ -0,0 +1,111 @@
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+ * @author Salesforce.org
+ * @date 2021
+ * @group
+ * @group-content ../../ApexDocContent/
+ * @description View Mapper class to to generate view models associated with
+ * Contact Language functionality.
+ */
+public virtual with sharing class ContactInformationSettingsVMapper {
+    /*********************************************
+     * @description Instance for Singleton Pattern
+     **********************************************/
+    @TestVisible
+    private static ContactInformationSettingsVMapper instance;
+
+    /*********************************************************************
+     * @description Empty constructor is not public for Singleton Pattern
+     **********************************************************************/
+    protected ContactInformationSettingsVMapper() {
+    }
+
+    /******************************************************************************
+     * @description Static method to get the current instance for Singleton pattern
+     * @return The instance of ContactInformationSettingsVMapper.
+     ******************************************************************************/
+    public static ContactInformationSettingsVMapper getInstance() {
+        if (instance == null) {
+            instance = new ContactInformationSettingsVMapper();
+        }
+
+        return instance;
+    }
+
+    /**********************************************************************************************
+     * @description Retrieve a model representing the Contact Language fluency settings specified
+     * in Hierarchy Settings and all available active options.
+     * @return A ContactLanguageSettingsVModel.
+     **********************************************************************************************/
+    public virtual ContactLanguageSettingsVModel getContactLanguageSettingsVModel() {
+        ContactLanguageSettingsModel contactLanguageSettingsModel = this.locateContactLanguageSettingsService()
+            .getContactLanguageSettingsModel();
+
+        Map<String, PicklistEntryModel> activeContactLanguageFluencyPicklistEntryModelsByName = this.locatePicklistEntryService()
+            .getActivePicklistEntryModelsByName(
+                this.locatePicklistEntryService()
```

Comment:
I'm going to call out that not using a local variable makes it easier to refactor code but we are trading off on detecting which method is throwing an exception if we get to debugging and encounter a problem.